ert3/engine/_run.py

```diff
@@ -137,15 +137,17 @@ def _load_ensemble_parameters(
         param.name: param.as_distribution() for param in parameters_config
     }
 
-    ensemble_parameters = {}
+    sensitivity_parameters = {}
     for input_record in ensemble.input:
         record_name = input_record.record
         record_source = input_record.source.split(_SOURCE_SEPARATOR)
-        assert len(record_source) == 2
-        assert record_source[0] == "stochastic"
-        parameter_group_name = record_source[1]
-        ensemble_parameters[record_name] = all_distributions[parameter_group_name]
-    return ensemble_parameters
+        if record_source[0] == "stochastic":
+            assert len(record_source) == 2
+            parameter_group_name = record_source[1]
+            sensitivity_parameters[record_name] = all_distributions[
+                parameter_group_name
+            ]
+    return sensitivity_parameters
 
 
 def _prepare_sensitivity(
```

Comment:
The implementation of `_prepare_sensitivity` is getting too long and with too high variance in granularity/detail level. The steps it does is:
- load sensitivity parameters,
- run sensitivity algorithm,
- start preparing experiment,
- move records in storage into experiment scope of storage and
- transpose and store sensitivity parameters in experiment scope of storage.

But the last two steps are intertwined and without encapsulation. It would probably read better if moved into two functions hinting towards the two last steps above.